sdk/examples/wallet/07_send_native_tokens.rs

```diff
@@ -40,12 +40,14 @@ async fn main() -> Result<()> {
             .set_stronghold_password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
             .await?;
 
-        let bech32_address = "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu".to_string();
+        let bech32_address =
+            Bech32Address::try_from_str("rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu").unwrap();
 
         let outputs = vec![AddressNativeTokens {
             address: bech32_address.clone(),
             native_tokens: vec![(*token_id, U256::from(10))],
-            ..Default::default()
+            return_address: Default::default(),
+            expiration: Default::default(),
```

Comment:
Thy this change? To show the the other fields can be set?

Reply:
No, I had to remove the `Default` impl.

Reply:
Ah because there is no default for Bech32Address and it actually didn't make sense before for the address field :+1: